src/applications/ivc-champva/10-7959C/containers/ConfirmationPage.jsx

```diff
@@ -81,24 +99,15 @@ export function ConfirmationPage(props) {
 
       {OverviewComp}
 
-      <h2 className="vads-u-font-size--h3">What to expect next</h2>
-      <p>
-        We'll contact you by mail or phone if we have questions or need more
-        information.
-        <br />
-        <br />
-        And we'll send you a letter in the mail with our decision.
-      </p>
       <div className="inset">
         <h3 className="vads-u-margin-top--0 vads-u-font-size--h4">
           Your submission information
         </h3>
-        {data.statementOfTruthSignature && (
+        {(data.statementOfTruthSignature || data.signature) && (
           <span className="veterans-full-name">
             <strong>Who submitted this form</strong>
             <br />
-            {data.statementOfTruthSignature}
-            <br />
+            {data.statementOfTruthSignature || data.signature}
           </span>
```

Comment:
Previously this was not accounting for keyname on custom signature field for this form.